test/integration/integration_test.go

```diff
@@ -161,6 +161,15 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`([^"]*) should be valid ([^"]*)$`,
 		shouldBeInValidFormat)
 
+	s.Step(`([^"]*) should match "([^"]*)"$`,
```

Comment:
add comment to indicate these are used with regex, as term 'match' is very generic and can be mistaken as equals.
or include 'expression' in the string or function name (I prefer function name)